ensemble/ctf/editor.py

```diff
@@ -65,7 +65,7 @@ def perform_with_color(self, event, color):
 
 
 class AddOpacityAction(BaseCtfEditorAction):
-    name = 'Add Opacity'
+    name = 'Add Opacity Node...'
```

Comment:
In GUI menus, an ellipsis indicates that a dialog or other window will pop up before the action is carried out. That's why there is a difference between these two action names.